pkg/util/sys/device_test.go

```diff
@@ -209,3 +215,17 @@ func TestParseUdevInfo(t *testing.T) {
 	m := parseUdevInfo(udevOutput)
 	assert.Equal(t, m["ID_FS_TYPE"], "ext2")
 }
+
+func TestListDevicesChildListDevicesChild(t *testing.T) {
+	executor := &exectest.MockExecutor{
+		MockExecuteCommandWithOutput: func(debug bool, actionName string, command string, arg ...string) (string, error) {
+			logger.Infof("action %s command %s", actionName, command)
+			return lsblkChildOutput, nil
+		},
+	}
+
+	device := "/dev/vdb"
+	child, err := ListDevicesChild(executor, device)
+	assert.NoError(t, err)
```

Comment:
Since this function can return partial successful results on error, I would really expect to test the error case when there are some disks valid.

Reply:
I'm not sure, do you want me to simulate a failure from `lsblk`?

Reply:
I think that makes sense? Like what happens if `lsblk` returns valid output but a nonzero exit code? We should probably make sure that this still counts as a failure.